client/src/components/timetable/ListCourse.jsx

```diff
@@ -1,14 +1,19 @@
 import React from "react";
+import { connect } from "react-redux";
+import { updateCurrentCourse } from "../../actions/UpdateCurrentCourse";
+
 const ListCourse = props => {
   return (
-    <ul className='courseSearch'>
+    <ul className="courseSearch">
       {Object.keys(props.courses).map(course => {
         return (
           <div
-            className='searchItem'
+            className="searchItem"
             key={course}
             id={course}
-            onClick={props.action}
+            onClick={() => {
+              props.action(course, props.courses);
```

Comment:
Maybe we could have a better name for this `action` thingy?

Reply:
ok